include/moq/publish_track_handler.h

```diff
@@ -126,7 +126,7 @@ namespace moq {
          *
          * @param metrics           Copy of the published metrics for the sample period
          */
-        virtual void MetricsSampled(const PublishTrackMetrics&& metrics);
+        virtual void MetricsSampled(PublishTrackMetrics metrics);
```

Comment:
Shouldn't this be const ref?
```suggestion
        virtual void MetricsSampled(const PublishTrackMetrics& metrics);
```